www/templates/session-detail.html

```diff
@@ -2,31 +2,35 @@
   <ion-content class="padding has-header">
     <div class="card">
       <div class="padding">
+        <img class="speaker-photo" ng-src="{{session.speakerphoto || 'https://i.imgflip.com/1slnr0.jpg'}}" />
         <div class="detail-container">
-          <div class="right-detail-section">
-            <h4 class="grey-text-session text-right">{{session.sessiontype}}</h4>
-            <h6 class="grey-text-session text-right">{{session.sessiontime}}</h6>
-          </div>
           <div class="left-detail-section">
             <h3 class="positive ">{{session.title}}</h3>
-              <div ui-sref="tab.speaker-detail({id:session.id})" class="session-detail-speaker">{{session.speakername}}
-                <i class="icon ion-chevron-right icon-accessory"></i>
-              </div>
+            <h4 ui-sref="tab.speaker-detail({id:session.id})" class="padding-top session-detail-speaker">{{session.speakername}}
+              <i class="icon ion-chevron-right icon-accessory"></i>
+            </h4>
```

Comment:
I moved this back into the single detail card. I think its easier for them to click on the bio link and map link if it is right under the title and photo